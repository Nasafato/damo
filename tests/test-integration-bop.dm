def test() : int {
	// Variable declarations come first
	int i;
	int j;
	num k;
	num l;	
	
	k = 1.0;
	l = 2.0;
	j = 2;
	i = 4;

	print_int( i/j );
	print_int( i + j );
	print_num(k*l);
	print_num(l-k);
	

	return 0;
}

test();