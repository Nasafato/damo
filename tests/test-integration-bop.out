2
6
2.000000
1.000000
